static/js/src/advantage/ecom-analytics-events.js

```diff
@@ -0,0 +1,58 @@
+export function addToCartEvent(product) {
+  dataLayer.push({
```

Comment:
You might want to null check dataLayer or try/catch in case the user has disabled analytics. We don't want this to lock up the UI scripts.